### PR TITLE
Add Debricked starter workflow

### PR DESCRIPTION
**Pull request by 4ernovm**
## Pre-requisites

- [x] Prior to submitting a new workflow, please apply to join the GitHub Technology Partner Program: [partner.github.com/apply](https://partner.github.com/apply?partnershipType=Technology+Partner).

---

### **Please note that at this time we are only accepting new starter workflows for Code Scanning. Updates to existing starter workflows are fine.**

---

## Tasks

**For _all_ workflows, the workflow:**

- [x] Should be contained in a `.yml` file with the language or platform as its filename, in lower, [_kebab-cased_](https://en.wikipedia.org/wiki/Kebab_case) format (for example, [`docker-image.yml`](https://github.com/actions/starter-workflows/blob/main/ci/docker-image.yml)).  Special characters should be removed or replaced with words as appropriate (for example, "dotnet" instead of ".NET").
- [x] Should use sentence case for the names of workflows and steps (for example, "Run tests").
- [x] Should be named _only_ by the name of the language or platform (for example, "Go", not "Go CI" or "Go Build").
- [x] Should include comments in the workflow for any parts that are not obvious or could use clarification.
- [x] Should specify least privileged [permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token) for `GITHUB_TOKEN` so that the workflow runs successfully.

**For _Code Scanning_ workflows, the workflow:**

- [x] Should be preserved under [the `code-scanning` directory](https://github.com/actions/starter-workflows/tree/main/code-scanning).
- [x] Should include a matching `code-scanning/properties/*.properties.json` file (for example, [`code-scanning/properties/codeql.properties.json`](https://github.com/actions/starter-workflows/blob/main/code-scanning/properties/codeql.properties.json)), with properties set as follows:
  - [x] `name`: Name of the Code Scanning integration.
  - [x] `creator`: Name of the organization/user producing the Code Scanning integration.
  - [x] `description`: Short description of the Code Scanning integration.
  - [x] `categories`: Array of languages supported by the Code Scanning integration.
  - [x] `iconName`: Name of the SVG logo representing the Code Scanning integration. This SVG logo must be present in [the `icons` directory](https://github.com/actions/starter-workflows/tree/main/icons).
- [x] Should run on `push` to `branches: [ $default-branch, $protected-branches ]` and `pull_request` to `branches: [ $default-branch ]`. We also recommend a `schedule` trigger of `cron: $cron-weekly` (for example, [`codeql.yml`](https://github.com/actions/starter-workflows/blob/c59b62dee0eae1f9f368b7011cf05c2fc42cf084/code-scanning/codeql.yml#L14-L21)).

### CODE REVIEW

**Review by Spiritaine — APPROVED** [review]

**Review by Spiritaine — APPROVED** [review]

File: code-scanning/debricked.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#####################################################################################################################################################################
+
+name: Debricked Scan
+
+on:
+  push:
+    branches: [ $default-branch, $protected-branches ]
+  schedule:
+    - cron: $cron-weekly
+
```
Comment (alexisabril) [inline code-scanning/debricked.yml:33]: Is there any impediment to restricting this job's privileges to the following?

```
permissions:
  contents: read
```

File: code-scanning/debricked.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+jobs:
+  vulnerabilities-scan:
```
Comment (alexisabril) [inline code-scanning/debricked.yml:35]:
```suggestion
  Vulnerabilities scan:
```

**Review by alexisabril — CHANGES_REQUESTED** [review]
Hi @4ernovm,

Thank you for the PR! I've added a few notes below, but I'm also looking to verify you as a TPP member. You checked the box that an application has been submitted, but I'm not seeing OpenText. Would you confirm if this application has been submitted and under which name?

**Comment by alexisabril** [comment]
@4ernovm, would you email our team at partnerships@github.com to verify your TPP membership? I want to make sure I identify the correct company and perhaps take this conversation outside of this issue.

File: code-scanning/debricked.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+name: Debricked Scan
+
+on: [push]
+
+permissions:
+  contents: read
+
+jobs:
+  vulnerabilities-scan:
```
Comment (jsoref) [inline code-scanning/debricked.yml:34]:
```suggestion
  vulnerabilities-scan:
    name: Vulnerabilities scan
```

File: code-scanning/debricked.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```
Comment (jsoref) [inline code-scanning/debricked.yml:38]:
```suggestion
      - uses: actions/checkout@v4
```

File: code-scanning/debricked.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+# 6. In the “Secrets” tab, click on “New repository secret”                                                                                                         #
+# 7. In the “Name” field, type the name of the secret                                                                                                               #
+# 8. In the “Secret” field, enter the value of the secret                                                                                                           #
+# 9. Click “Add secret”                                                                                                                                             #
+# 10. You should now be ready to use the workflow!                                                                                                                  #
+#####################################################################################################################################################################
+
+name: Debricked Scan
+
+on: [push]
```
Comment (jsoref) [inline code-scanning/debricked.yml:28]: Personal preference:
```suggestion
on:
  push:
```

This makes it easier for people to make changes to the workflow w/o taking blame for lines that they aren't conceptually changing.

File: code-scanning/debricked.yml (hunk 2 above)
Comment (alexisabril) [reply code-scanning/debricked.yml:35]: Ah, fair callout! This is passing CI as-is, so I'll resolve this conversation.

**Review by alexisabril — APPROVED** [review]

File: code-scanning/debricked.yml (hunk 2 above)
Comment (jsoref) [reply code-scanning/debricked.yml:35]: You can apply this instead https://github.com/actions/starter-workflows/pull/2107/files#r1580129334

**Pull request merged**